src/lightmap.cpp

```diff
@@ -571,44 +568,60 @@ void map::build_seen_cache()
             //
             // The naive solution of making the mirrors act like a second player
             // at an offset appears to give reasonable results though.
```

Comment:
That would mean more map accesses in the very expensive castlight method,
in reality we expect to only have very few of them present, so it's far
cheaper to enumerate them directly.  The thing about the implementation
being naive is referring to the fact that it doesn't 'bounce' off the
mirror, instead it radiates in all directions.  Ideally we'd calculate an
angle of incidence and only cast in the direction indicated by the
reflection.